src/images/icons.json

```diff
@@ -1,114 +1,117 @@
 {
-  "icons": [
-    "active-task",
-    "archive",
-    "arrow-wallet",
-    "arrow-rotate-ccw",
-    "arrow-right",
-    "at-sign-circle",
-    "attachment",
-    "bell",
-    "caret-down-small",
-    "caret-left",
-    "caret-right-small",
-    "caret-right",
-    "caret-up-small",
-    "check-mark",
-    "checklist",
-    "circle-add",
-    "circle-archive",
-    "circle-arrow-back",
-    "circle-arrow-down",
-    "circle-back",
-    "circle-check",
-    "circle-close",
-    "circle-eye",
-    "circle-person",
-    "circle-plus",
-    "circle-minus",
-    "circle-thumbs-down",
-    "circle-thumbs-down-grey",
-    "circle-thumbs-down-outlined",
-    "circle-thumbs-up",
-    "circle-thumbs-up-grey",
-    "circle-thumbs-up-outlined",
-    "clny-token",
-    "close",
-    "colony-logo",
-    "comment",
-    "cup",
-    "drag-handle",
-    "emoji-alarm-lamp",
-    "emoji-balance",
-    "emoji-bag-money-sign",
-    "emoji-building",
-    "emoji-calendar",
-    "emoji-clipboard",
-    "emoji-crane",
-    "emoji-dna",
-    "emoji-dollar-stack",
-    "emoji-edit-tools",
-    "emoji-firebolt",
-    "emoji-gold-coin",
-    "emoji-goblin",
-    "emoji-judge",
-    "emoji-medal",
-    "emoji-money-wings",
-    "emoji-padlock",
-    "emoji-padlock-closed",
-    "emoji-party",
-    "emoji-pen",
-    "emoji-pencil-note",
-    "emoji-piggy-bank",
-    "emoji-seed-sprout",
-    "emoji-shooting-star",
-    "emoji-smiley-nerd",
-    "emoji-strong-person",
-    "emoji-superman",
-    "emoji-world-globe",
-    "emoji-yellow-superman",
-    "empty-task",
-    "envelope",
-    "file",
-    "filled-circle-person",
-    "forbidden-signal",
-    "four-square",
-    "graph-line",
-    "trash",
-    "line-arrow-circle",
-    "lock",
-    "no-task",
-    "pencil",
-    "person",
-    "phrase",
-    "question-mark",
-    "question-mark-inverted",
-    "return-arrow",
-    "search",
-    "settings",
-    "sign-off",
-    "square-plus",
-    "star",
-    "tag",
-    "three-dots-row",
-    "triangle-warning",
-    "unlock",
-    "user-plus",
-    "wallet"
-  ],
-  "multiColorIcons": [
-    "circle-check-primary",
-    "metamask",
-    "shield-pink"
-  ],
-  "badges": [
-    "Alessandra Aphid",
-    "Terry Termite",
-    "Ava Ant",
-    "Bong Joon Bee",
-    "Werner Wasp",
-    "Berthold Beetle",
-    "Sanjeet Shrimp",
-    "Norman Naked Mole Rat"
-  ]
+    "icons": [
```

Comment:
Related to #96, this is what my prettier does to this file and we should probably commit a simple prettier config to the repo so it doesnt get changed back by accident.